extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -521,4 +558,39 @@ private void ensureMaxRowsInAWindowConstraint(int numRowsInWindow)
       ));
     }
   }
+
+  /**
+   * Increments the value of the partition boosting column. It should be called once the row value has been written
+   * to the frame
+   */
+  private void incrementBoostColumn()
+  {
+    partitionBoostVirtualColumn.setValue(partitionBoostVirtualColumn.getValue() + 1);
```

Comment:
inline back

Reply:
I had created the method as we had the line at 2 places now instead of 1.